src/cloudWatchLogs/changeLogSearch.ts

```diff
@@ -74,42 +75,38 @@ export async function changeLogSearchParams(
     registry: LogDataRegistry,
     param: 'filterPattern' | 'timeFilter'
 ): Promise<void> {
-    const oldUri = getActiveDocumentUri(registry)
-    if (!registry.isRegistered(oldUri)) {
-        telemetry.cloudwatchlogs_filter.emit({
-            result: 'Failed',
-            source: 'Editor',
-            cloudWatchResourceType: isLogStreamUri(oldUri) ? 'logStream' : 'logGroup',
-            hasTimeFilter: param === 'timeFilter',
-            hasTextFilter: param === 'filterPattern',
-        })
-        throw new Error(`cwl: Unable to find data for active URI ${oldUri}`)
-    }
-
-    const oldData = registry.getRegisteredLog(oldUri)
-    const newData = await getNewData(param, oldData)
+    telemetry.cloudwatchlogs_open.run(async span => {
+        span.record({ source: 'Editor' })
+        const oldUri = getActiveDocumentUri(registry)
+        if (!registry.isRegistered(oldUri)) {
+            telemetry.cloudwatchlogs_filter.record({
+                source: 'Editor',
```

Comment:
Should `result` be included in this `telemetry.cloudwatchlogs_filter.record()` call?